src/rgw/rgw_lua.cc

```diff
@@ -5,11 +5,8 @@
 #include "rgw_lua_utils.h"
 #include "rgw_sal_rados.h"
 #include "rgw_lua.h"
-#ifdef WITH_RADOSGW_LUA_PACKAGES
```

Comment:
now that the luarock code is in this file, we need the WITH_RADOSGW_LUA_PACKAGES macros

Reply:
okay